build-system/server/new-server/transforms/README.md

```diff
@@ -5,7 +5,8 @@
 **Structure:** Each subdirectory should contain the following:
 
 - A single `.ts` file that implements a specific transform (`*-transform.ts`)
-- A `test/` subdirectory containing one or more pairs of input and output files (`test/*-[input|output].html`)
+- A `test/` subdirectory containing one or more subdirectories labelled by test name, with each subdirectory containing one pair of input and output files (`test/*/[input|output].html`)
```

Comment:
ideally we'd allow for any number of input/output pair in the subdirectory

Reply:
Is there an example in the Babel plugin tests that has multiple pairs of inputs in a subdirectory?